src/components/FileInputButton.js

```diff
@@ -0,0 +1,56 @@
+import {Button} from '@sanity/ui'
+import {uniqueId} from 'lodash'
+import React from 'react'
+import styled from 'styled-components'
+
+const HiddenInput = styled.input`
+  overflow: hidden;
+  width: 0.1px;
+  height: 0.1px;
+  opacity: 0;
+  position: absolute;
+  z-index: -1;
+`
+
+const Label = styled.label`
+  position: relative;
+`
+
+const handleSelect = ({event, onSelect}) => {
+  if (onSelect) {
+    onSelect(event.target.files)
+  }
+}
+
+const handleButtonClick = (inputRef) => {
+  if (inputRef && inputRef.current) {
+    inputRef.current.click()
+  }
+}
+
+export const FileInputButton = ({onSelect, accept = 'video/*', type = 'file', ...props}) => {
```

Comment:
This isn't a new change, but accept `accept="video/*"` on input file types doesn't always behave reliably. I ran into this here, we might want to consider removing the `accept` prop:

https://github.com/contentful/apps/pull/91#issuecomment-638903762

Reply:
Removed in 4bfb986